docs/EN_US/ECLLanguageReference/ECLR_mods/Recrd-DATASET.xml

```diff
@@ -341,6 +351,14 @@
   <para>The ninth form is used to create a DATASET using a TRANSFORM function
   (see <emphasis role="bold">DATASET from TRANSFORM</emphasis> below)</para>
 
+  <para>The __OPTION__(LEGACY) option can be used if your dataset definition
```

Comment:
Why is this here?  Most of the other optional options e.g. distributed, local, do not have any text in this section.  This section is describing the different format of the DATASET command.